### PR TITLE
add 'make package' libadb target to make libadb packages

### PR DESCRIPTION
**Pull request by mykmelez**
@ochameau, can you take a look at this? It should be straightforward: a `make package` target for libadb (so we don't have to construct these packages by hand).

### CODE REVIEW

File: android-tools/adb-bin/Makefile [hunk 1]
```diff
@@ -3,15 +3,24 @@
...
 ifeq ($(OS), Windows_NT)
   VS_DEVELOPER_CMD_PATH = vcvars32.bat
   ADB_TARGETS = winlib
   WINDOWS_ARTIFACTS = ../win-out vc110.idb vc110.pdb ../adb-win-api/api/build*_w*_x86.* \
       ../adb-win-api/api/obj* ../adb-win-api/api/*.idb ../adb-win-api/api/*.pdb \
       ../adb-win-api/winusb/obj* ../adb-win-api/winusb/build*_w*_x86.*
+  PACKAGE_FILES += \
+    ../win-out/libadb.dll \
+    ../adb-win-api/api/objfre_wxp_x86/i386/AdbWinApi.dll \
+    ../adb-win-api/winusb/objfre_wxp_x86/i386/AdbWinUsbApi.dll
```
Comment (ochameau) [inline android-tools/adb-bin/Makefile]: Wouldn't it be better to use `obj*` instead of `objfre_wxp_x86`, like what is being done for `WINDOWS_ARTIFACTS`?

**Comment by ochameau** [comment]
Ship it!

**Comment by mykmelez** [comment]
> Wouldn't it be better to use obj\* instead of objfre_wxp_x86, like what is being done for WINDOWS_ARTIFACTS?

Yes, it would! Fixed in ec74058c6050cf98d74009d262dde74fd66910b1.

I also fixed a couple other issues:
- Use the `-j` flag to _zip_ to trim directory paths from the files being packaged (Windows-specific, as only Windows has multiple source dirs and builds into separate object dirs).
- Build the same thing (libadb) by default on all three platforms.

**Comment by mykmelez** [comment]
> Ship it!

À votre demande!

**Pull request merged**